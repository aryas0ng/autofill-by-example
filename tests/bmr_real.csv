age_sv,height,weight,gender,BMR
52,165,49.5,female,1191.3295
46,156,56.25,female,1251.84475
52,169,55.79,female,1261.88513
50,174,69.2,female,1410.0374
52,167,55.79,female,1255.68913
52,180,70.42,female,
51,164,68.49,female,
53,154,77.56,female,
47,171,54.2,female,
47,151.5,83.55,female,
52,160,53.5,female,
48,162,61.8,female,
50,156,49.8,female,
45,169,59.8,female,
51,166,88.9,female,
47,175,80.73,female,
45,166,73.9,female,
49,161,56.7,female,
54,170,70.3,female,
49,164,66.9,female,
49,172,73.48,female,
45,152,51.94,female,
47,166.5,60.5,female,
48,165,63.73,female,
52,163,78.9,female,
49,164,50.08,female,
48,161,68,female,
51,171.5,91,female,
49,170,63.5,female,
51,163.5,60.96,female,
51,161,66.2,female,
47,164,80.78,female,
51,171.5,55.8,female,
50,162.5,64,female,
49,175.5,57,female,
45,156.5,50.3,female,
50,162,67.49,female,
48,171,67.1,female,
46,165,68.9,female,
50,171,58.9,female,
49,156,65.8,female,
45,154,51.3,female,
53,171.5,70.8,female,
51,166,73.5,female,
49,164,89.8,female,
54,163,78.6,female,
51,165,56.7,female,
53,163,66.7,female,
54,165,93.44,female,
48,173,73.9356,female,
46,154,57.15,female,
,,,,
45,166,64.18,female,
52,160,61.69,female,
43,155,98.88,female,