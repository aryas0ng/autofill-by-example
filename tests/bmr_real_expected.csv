age_sv,height,weight,gender,BMR
52,165,49.5,female,1191.3295
46,156,56.25,female,1251.84475
52,169,55.79,female,1261.88513
50,174,69.2,female,1410.0374
52,167,55.79,female,1255.68913
52,180,70.42,female,1431.24674
51,164,68.49,female,1368.16203
53,154,77.56,female,1412.39232
47,171,54.2,female,1275.0284
47,151.5,83.55,female,1486.01685
52,160,53.5,female,1212.8275
48,162,61.8,female,1313.0936
50,156,49.8,female,1174.8816
45,169,59.8,female,1329.2756
51,166,88.9,female,1563.0893
47,175,80.73,female,1532.74331
45,166,73.9,female,1450.3643
49,161,56.7,female,1258.5059
54,170,70.3,female,1390.4971
49,164,66.9,female,1362.1193
49,172,73.48,female,1447.74856
45,152,51.94,female,1203.92818
47,166.5,60.5,female,1319.3435
48,165,63.73,female,1340.23431
52,163,78.9,female,1456.9953
49,164,50.08,female,1206.58476
48,161,68,female,1367.327
51,171.5,91,female,1599.547
49,170,63.5,female,1349.2675
51,163.5,60.96,female,1296.98312
51,161,66.2,female,1337.6924
47,164,80.78,female,1499.12766
51,171.5,55.8,female,1274.0526
50,162.5,64,female,1326.326
49,175.5,57,female,1306.201
45,156.5,50.3,female,1202.7041
50,162,67.49,female,1357.04903
48,171,67.1,female,1389.9847
46,165,68.9,female,1396.7013
50,171,58.9,female,1305.4993
49,156,65.8,female,1327.1636
45,154,51.3,female,1204.2061
53,171.5,70.8,female,1404.0976
51,166,73.5,female,1420.6855
49,164,89.8,female,1573.8756
54,163,78.6,female,1445.5612
51,165,56.7,female,1262.2379
53,163,66.7,female,1339.8519
54,165,93.44,female,1588.98268
48,173,73.9356,female,1459.389493
46,154,57.15,female,1253.97105
45,166,64.18,female,1360.48346
52,160,61.69,female,1288.56043
43,155,98.88,female,1655.93636